example/spec/select

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+if [ ! -z "${CIRCLE_USERNAME}" ]; then
+    declare -A developers
+    developers=( ["abarciauskas-bgse"]="aimee-test" \
```

Comment:
Should we put something in the README about this?

Reply:
yeah, we can.